### PR TITLE
add search score results

### PR DESCRIPTION
**Pull request by sobirjonovs**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | no
| Tests pass?   | no
| Fixed issues  | no issue

### CODE REVIEW

**Comment by sobirjonovs** [comment]
I needed it to get overall scores and each record scoring results in the elasticsearch result list.

**Comment by sobirjonovs** [comment]
I needed it to get overall scores and each record scoring results in the elasticsearch result list.

**Review by pollux83 — APPROVED** [review]

**Comment by samdark** [comment]
Need a line for CHANGELOG and, ideally, some unit tests confirming behavior.

File: Query.php [hunk 1]
```diff
@@ -246,7 +246,9 @@ public function all($db = null)
         if ($this->emulateExecution) {
             return [];
         }
-        $result = $this->createCommand($db)->search();
+        $result = $this->createCommand($db)->search([
+            'track_scores' => 'true'
```
Comment (lubosdz) [inline Query.php:250]: This is no good solution - pls don't hard code your own one single case. Tracking scores adds extra overhead which may not be OK for others. You already have `$options` that might be supplied along into query e.g. `$result = $this->createCommand($db)->search($this->options)` or add a new flag e.g. `public $trackScores = false`. Unit test?

Comment (beowulfenator) [reply Query.php:250]: I agree, let's not hardcode it.

**Comment by beowulfenator** [comment]
I am rejecting this PR because it is not generic enough. If in the future somebody is inclined to make a proper implementation of this concept (i.e. a "track scores" flag in Command) they are welcome to do so.

**Pull request closed without merging**